<launch>
  <arg name="output" default="screen"/>
  <arg name="bag" default=""/>
  <arg name="detector_type" default="Umich"/>
  <arg name="start_time" default="0.0"/>
#	launch-prefix="gdb -ex run --args"
  <node pkg="tagslam" type="sync_and_detect_node" name="sync_and_detect"
	output="$(arg output)" clear_params="True">
    <rosparam param="image_topics"> ["/fla/ovc_node/left/image_raw", "/fla/ovc_node/right/image_raw"]</rosparam>
    <rosparam param="tag_topics"> ["/fla/ovc_node/left/tags", "/fla/ovc_node/left/tags"]</rosparam>
    <param name="bag_file" value="$(arg bag)"/>
    <param name="detector_type" value="$(arg detector_type)"/>
    <param name="black_border_width" value="1"/>
    <param name="start_time" value="$(arg start_time)"/>
  </node>
</launch>
